cmake/Platform/Common/Install_common.cmake

```diff
@@ -450,6 +450,13 @@ function(ly_setup_runtime_dependencies)
     # Common functions used by the bellow code
     install(CODE
 "function(ly_copy source_file target_directory)
+    if(\"\${source_file}\" MATCHES \"\\\\.[Ff]ramework[^\\\\.]\")
+
+        # fixup origin to copy the whole Framework folder
+        string(REGEX REPLACE \"(.*\\\\.[Ff]ramework).*\" \"\\\\1\" source_file \"\${source_file}\")
+        get_filename_component(target_filename \"\${source_file}\" NAME)
+
+    endif()
```

Comment:
This isn't needed for any platforms other than Mac(and maybe iOS), but it also doesn't break anything on other platforms. Should I still create a platform specific version of this install(CODE) block for Mac?

Reply:
an easy one is to just define the copy function in a variable and make Mac define it to a different value
it would be weird to be doing those sort of checks in windows, I agree it shouldnt produce any issues, but we have the mechanics to do it per platform